docs/adr/0008-accept-unrecognised-fields.md

```diff
@@ -0,0 +1,44 @@
+#  Accept metadata that includes unrecognized fields
+
+- Status: accepted
+- Date: 2021-04-08
+
+Technical Story: https://github.com/theupdateframework/tuf/issues/1266
+
+## Context and Problem Statement
+
+The current reference implementation will ignore unrecognized fields in a
+metadata file when loading it.
+This leads to the side effect that if you read a metadata file with unrecognized
+fields and immediately write it back to the disk, this file will be modified.
+
+Furthermore, some TAPs ([6](https://github.com/theupdateframework/taps/blob/master/tap6.md), [10](https://github.com/theupdateframework/taps/blob/master/tap10.md), [14](https://github.com/theupdateframework/taps/blob/master/tap14.md), [15](https://github.com/theupdateframework/taps/blob/master/tap15.md), and [16](https://github.com/theupdateframework/taps/blob/master/tap16.md)) are relying on that unrecognized
+fields will be accepted to introduce new fields to the specification without
+making the metadata invalid for older clients which don't recognize the field.
+
+## Decision Drivers
+- The TUF specification implies support for custom attribute-value fields
+See [Document formats](https://theupdateframework.github.io/specification/latest/#document-formats)
+- Backward compatibility
+- If we perform the following operations on a metadata file with no
+intermediate operations:
+1. read the metadata file
+2. write the metadata file back to the disk
+the checksum (the content) of the file must not be changed.
```

Comment:
this Markdown is rendered such that "the checksum (the content) of the file must not be changed" appears on the same line as "write the metadata file back to the disk"